src/importexport/musicxml/tests/data/testBreathMarks.xml

```diff
@@ -59,7 +59,7 @@
         <voice>1</voice>
         <notations>
           <articulations>
-            <breath-mark>comma</breath-mark>
+            <breath-mark placement="above">comma</breath-mark>
```

Comment:
I guess, if this is the default placement it shouldn't be here.

Reply:
They are getting written (exported) with that property explicitly set, for them to import cleanly into Mu3 and ealier, which doesn't have the import fix of this PR

Reply:
I agree with @cbjeukendrup that we shouldn't try to build the export here around the needs for import somewhere else. I do not care too much about this attribute in particular, but it looks to me if we're about to open the gates of hell.

Reply:
This one is pretty harmless IMHO, we just write a bit more than absolutely needed.

Reply:
But see https://github.com/musescore/MuseScore/pull/20370/commits/e334a8cbd5f482e3b2d02dccf85488290ec2a827#r1484525120